app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -327,3 +314,52 @@ msgstr "Cyfeirnod"
 #: app/templates/tasklist.html:94
 msgid "Status:"
 msgstr "Statws:"
+
+#: app/templates/partials/available_in_language.html:2
+#, python-format
+msgid ""
+" This service is also available <a href=\"%(url)s\" class=\"govuk-"
+"link\">in Welsh (Cymraeg)"
+msgstr ""
+"Mae'r gwasanaeth yma hefyd ar gael <a href=\"%(url)s\" class=\"govuk-"
+"link\">yn Saesneg (English)"
+
+#: app/templates/partials/available_in_language.html:9
+msgid "You cannot change the language of an existing application."
+msgstr ""
+"Ni fyddwch yn gallu newid iaith unwaith y byddwch chi wedi dechrau eich "
+"cais."
+
+#: app/templates/partials/contact_details.html:3
+msgid "Email:"
+msgstr "Ebost:"
+
+#: app/templates/partials/contact_details.html:7
+#, fuzzy
```

Comment:
I think we should get rid of fuzzy translations - they don't show up correctly when running the app - just delete the `fuzzy` line